maintainers/maintainer-list.nix

```diff
@@ -1333,11 +1333,6 @@
     github = "dmjio";
     name = "David Johnson";
   };
-  dmvianna = {
```

Comment:
I meant to just a separate commit in this pr.

Reply:
Sorry. It is now merged in #63071.